juju/charm.py

```diff
@@ -1,62 +1,69 @@
 from juju.framework import Object, Event, EventBase, EventsBase
 
 
-class InstallEvent(EventBase):
+class JujuEvent(EventBase):
```

Comment:
Agree with @dshcherb in the sense we probably don't need this one. I mean, we have events, and events coming from hooks, but since this is all juju every other event is in theory a juju event as well.

But maybe I'm missing something?